### ISSUE TITLE
linter-flake8 does not work after update linter to 2.3

### ISSUE THREAD

**Issue by harryln** [AtomLinter/linter-flake8#669]
Dear Sir/Madam,

I have found out that linter-flake8 does not work after updating linter from 2.2 to 2.3 for both Windows and Linux. I have no idea it is linter's problem or anything else.

**Comment by tcbegley** [comment]
I had the same issue. Running `Linter: Debug` in the command palette seems to suggest that linter is not detecting linter-flake8, but I don't know how to diagnose further:

```
Platform: darwin
Atom Version: 1.34.0
Linter Version: 2.3.0
Opened file is ignored: No
Matching Linter Providers: 
  - isort
Disabled Linter Providers: 
Standard Linter Providers: 
  - isort
Indie Linter Providers: 
  - Prettier
UI Providers: 
  - Linter
Ignore Glob: **/*.min.{js,css}
VCS Ignored Paths are excluded: true
Current File Scopes: 
  - *
  - source.python
  - keyword.control.import.python
```

I managed to resolve for now by rolling back to older versions of linter and linter-flake8, here's what I did in case it's useful to anyone with the same issue:

```
apm uninstall linter linter-flake8
apm install linter@2.2.0 linter-flake8@2.2.0
```

**Comment by harryln** [comment]
@tcbegley, many thanks. Based on your solution, I have only done:
```
apm uninstall linter
apm install linter@2.2.0
```
`linter-flake8` works again. Many thanks for your prompt answer.

**Comment by Arcanemagus** [comment]
Duplicate of #665
